light/stateprovider.go

```diff
@@ -380,69 +392,43 @@ func (s *StateProviderP2P) consensusParams(ctx context.Context, height int64) (t
 
```

Comment:
Can you also fix the comments for this function?

Reply:
Comment was updated @yzang2019